presto-orc/src/test/java/com/facebook/presto/orc/TestJniZstdDecompression.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import com.facebook.presto.orc.zstd.ZstdJniCompressor;
+import com.facebook.presto.orc.zstd.ZstdJniDecompressor;
+import com.facebook.presto.testing.assertions.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Random;
+
+public class TestJniZstdDecompression
+{
+    private static final long MAX_BUFFER_SIZE = 4 * 1024L * 1024 * 1024;
+    private final ZstdJniCompressor compressor = new ZstdJniCompressor();
+    private final ZstdJniDecompressor decompressor = new ZstdJniDecompressor(new OrcDataSourceId("test"), MAX_BUFFER_SIZE);
+
+    @Test
+    public void testDecompression()
+            throws OrcCorruptionException
+    {
+        byte[] srcBytes = generateRandomBytes();
+        byte[] compressedBytes = new byte[1024 * 1024];
+        int size = compressor.compress(srcBytes, 0, srcBytes.length, compressedBytes, 0, compressedBytes.length);
+        int outputSize = decompressor.decompress(compressedBytes, 0, size,
```

Comment:
Also, we enforce one parameter per line

Reply:
@gauravkm Gaurav, you can generate ORC files on the fly. See com.facebook.presto.orc.OrcTester#writeOrcColumnHive